config/routes.rb

```diff
@@ -11,4 +11,13 @@
 
   # Defines the root path route ("/")
   # root "posts#index"
+  namespace :api do
+    namespace :v1 do
```

Comment:
I don't see any primary `index` view here...what is going to drive your main page that lists multiple users/schedules?